proto/cacheclient.proto

```diff
@@ -139,6 +139,16 @@ message _DictionarySetRequest {
 
 message _DictionarySetResponse {}
 
+message _IncrementRequest {
```

Comment:
I don't see the `rpc` definition in Scs above. Is that intended to come later?

nit: These types are sandwiched between some dictionary types. Did you consider putting them maybe up by the other scalar definitions or down at the bottom instead?

Reply:
🤦 yep, both of those concerns were intended to be fixed before I pushed.

Reply:
Fixed in [fff7562](https://github.com/momentohq/client-protos/pull/106/commits/fff7562676df26d8f28a1a9443b1289beb469f85), thanks for catching those two issues!